win32/appveyor.bat

```diff
@@ -194,12 +194,19 @@ copy COPYING package\license > nul
 copy win32\mkstemp\COPYING.MinGW-w64-runtime.txt package\license > nul
 robocopy man package\man *.html > nul
 cd package
-7z a ..\ctags-%ver%-%ARCH%.debug.zip %filelist% %dirlist%
-strip *.exe
+call :strip ctags.exe
+call :strip readtags.exe

```

Comment:
Doesn't "call :string *.exe" work?
With "call :string *.exe", we don't have to update win32/appveryor.bat when I add "edittags.exe" or some new commands.

Reply:
I doesn't work, but we can use the "for" command to expand wildcards.
I will update.

Reply:
updated